docs/reference/commandline/network_create.md

```diff
@@ -14,18 +14,111 @@ parent = "smn_cli"
 
     Creates a new network with a name specified by the user
 
-      -d, --driver=      Driver to manage the Network
-      --help=false       Print usage
+    --aux-address=map[]      Auxiliary ipv4 or ipv6 addresses used by network driver
+    -d --driver=DRIVER       Driver to manage the Network bridge or overlay. The default is bridge.
+    --gateway=[]             ipv4 or ipv6 Gateway for the master subnet
+    --help=false             Print usage
+    --ip-range=[]            Allocate container ip from a sub-range
+    --ipam-driver=default    IP Address Management Driver
+    -o --opt=map[]           Set custom network plugin options
+    --subnet=[]              Subnet in CIDR format that represents a network segment
 
-Creates a new network that containers can connect to. If the driver supports multi-host networking, the created network will be made available across all the hosts in the cluster. Daemon will do its best to identify network name conflicts. But its the users responsibility to make sure network name is unique across the cluster. You create a network and then configure the container to use it, for example:
+Creates a new network. The `DRIVER` accepts `bridge` or `overlay` which are the built-in network drivers. If you have installed a third party or your own custom network driver you can specify that `DRIVER` here also. If you don't specify the `--driver` option, the command automatically creates a `bridge` network for you.
 
+When you install Docker Engine it creates a `bridge` network automatically.
```

Comment:
@mavenugo First can you confirm my ask -- does `bridge` support links where other `bridge` type networks do not? That is a fairly important points for users.

My concern about calling it the "default bridge" network everywhere is there are a few situations where default comes into play WRT to the keyword bridge

* when docker starts up it creates a default network called `bridge` 
* the network named `bridge` does not have the same behavior of the network created with the `bridge` driver in that it supports link
* the name of the `bridge` network docker creates by default is `bridge` 
* if a user doesn't specify a driver, then `bridge`  Docker uses the bridge driver by default

If a user does an ifconfig to look at their network stack the default bridge network appears as `bridge0` and this is rather important to keep in mind. It comes naturally to you but not to readers.  And less so as they work with the tool as `bridge` is now overloaded given the implementation. In fact, now that I've written it all out, I might create a section "When is a bridge not a bridge" :-D